x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/analysis/AnalyzerContext.java

```diff
@@ -52,7 +53,7 @@ public AnalyzerContext(Configuration configuration, EsqlFunctionRegistry functio
         this(
             configuration,
             functionRegistry,
-            result.indices(),
+            result.indexResolutions(),
```

Comment:
Again, we have singular/plural confusion here - should we consistently use plural for collections and singular for single item?